tests/unit/test_ansible_inventory.py

```diff
@@ -89,7 +89,7 @@ class TestAnsibleInventory:
     def test_layouts(self, layout, db, metadata):
 
         config = provisioning_config(layout)
-        ans_inv = AnsibleInventoryOutput(config, db, metadata)
+        ans_inv = AnsibleInventoryOutput(config, db, metadata, None)
```

Comment:
Relevant comment: https://github.com/neoave/mrack/pull/58/files#r529522230
nit-pick: personally I don't like to see `None` passed as positional arguments, doesn't look good.

Reply:
```suggestion
        ans_inv = AnsibleInventoryOutput(config, db, metadata, path=None)
```